mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -24,6 +24,11 @@ def cross_entropy(pred,
         ignore_index=ignore_index)
 
     # apply weights and do the reduction
+    # pytorch's official cross_entropy average loss over non-ignored elements
+    # refer to https://github.com/pytorch/pytorch/blob/56b43f4fec1f76953f15a627694d4bba34588969/torch/nn/functional.py#L2660  # noqa
+    if avg_factor is None:
+        avg_factor = label.numel() - (label == ignore_index).sum().item()
```

Comment:
the `avg_factor` will be zero if all elements in `label` is `ignore_index`, and in `weight_reduce_loss` function, `loss.sum() / avg_factor` will cause `nan`

Reply:
Hi, thanks for reviewing the PR. Since I'm no longer at SenseTime, please feel free to close this PR and recreate a fix. Thanks